src/Nethermind/Nethermind.Evm/CodeAnalysis/ByteCodeValidator.cs

```diff
@@ -0,0 +1,52 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Nethermind.Logging;
+using Nethermind.Core.Attributes;
+using Nethermind.Core.Specs;
+using Org.BouncyCastle.Crypto.Agreement.Srp;
+
+namespace Nethermind.Evm.CodeAnalysis
+{
+    internal static class ByteCodeValidator
+    {
+        private static EvmObjectFormat? EofFormatChecker = new EvmObjectFormat();
+        public static void Initialize(ILogger logger = null)
```

Comment:
any good reasons for this being static class? We could move initialization to constructor

Reply:
It was static prior to adding the Logger, will try fix it asap.